graphics/libpixman-devel/Portfile

```diff
@@ -42,7 +45,11 @@ depends_build-append \
 # clang on Xcode 4.1 cannot build libpixman
 compiler.blacklist      llvm-gcc-4.2 {clang < 211.10.1}
 
-configure.args          ac_cv_prog_AWK=/usr/bin/awk \
+configure.checks.implicit_function_declaration.whitelist-append \
+                        strchr
+
+configure.args-append \
```

Comment:
Ryan, while I tried not to change too much, was hoping you'd be okay with using `configure.args-append`?

If not, then no worries... and happy to change it back.